src/Audit/Audit.php

```diff
@@ -189,21 +202,26 @@ public function log(string $userInternalId, string $userId, string $event, strin
      * Get All Logs By User ID.
      *
      * @param  string  $userId
-     * @param  int  $limit
-     * @param  int  $offset
+     * @param  int|null  $limit
+     * @param  int|null  $offset
      * @param  Document|null  $orderAfter
-     * @return Document[]
+     * @return array<Document>
+     *
+     * @throws \Exception
      */
-    public function getLogsByUser(string $userId, int $limit = 25, int $offset = 0, Document $orderAfter = null): array
+    public function getLogsByUser(string $userId, ?int $limit = null, ?int $offset = null, ?Document $orderAfter = null): array
     {
-        /** @var Document[] $result */
+        /** @var array<Document> $result */
         $result = Authorization::skip(function () use ($userId, $limit, $offset, $orderAfter) {
-            $queries = $this->buildQuery(['userId' => $userId], Query::TYPE_EQUAL);
+            $queries[] = Query::equal('userId', [$userId]);
 
-            $queries[] = Query::limit($limit);
-            $queries[] = Query::offset($offset);
-            $queries[] = Query::orderDesc('');
-            if ($orderAfter) {
+            if (! \is_null($limit)) {
+                $queries[] = Query::limit($limit);
+            }
+            if (! \is_null($offset)) {
+                $queries[] = Query::offset($offset);
+            }
+            if (! \is_null($orderAfter)) {
                 $queries[] = Query::cursorAfter($orderAfter);
```

Comment:
Is Cursor intentionally , not "Order"?

Reply:
Yep, `$orderAfter` is just the parameter name for the cursor document